src/Components/Body/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,16 @@
+import './ErrorBoundary.scss'
+
+type ErrorBoundaryProps = {
+    children: React.ReactNode
+}
+
+export default function ErrorBoundary(props: ErrorBoundaryProps) {
+    const ErrorText = () => (
```

Comment:
I would create separate component. Something like ErrorView. It would be useful in next homeworks